src/index.js

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import CalculatorDisplay from './components/Calculator';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <CalculatorDisplay />
```

Comment:
- [ ] great job so far. Kindly use the app component in the index.js file.
- [ ] import the calculatordisplay class in the app.js component
- [ ] please ensure the app.js component is class based. display it from there. check the following example for you app.js component.
```
class App extends Component {
  render() {
    // reference as a js object
    return {
      <Calculatordisplay />
    };
  }
}
```